packages/xrpl/src/models/transactions/DIDSet.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  BaseTransaction,
+  isString,
+  validateBaseTransaction,
+  validateOptionalField,
+} from './common'
+
+/**
+ * @category Transaction Models
+ */
+export interface DIDSet extends BaseTransaction {
+  TransactionType: 'DIDSet'
+
+  Data?: string
+
+  DIDDocument?: string
```

Comment:
Should these be documented?

Reply:
There's no documentation yet, I will add it when it's available. But that won't be for another couple of weeks.